docs/observability/ovn-observability.md

```
@@ -37,16 +38,28 @@ insights of what ovn-kubernetes is doing with a packet and why.
 
 To enable this feature, use `--observability` flag with `kind.sh` script or `--enable-observability` flag with `ovnkube` binary.
 
-To see the samples, use `ovnkube-observ` binary, use `-h` to see allowed flags.
+To see the samples, use `ovnkube-observ` binary, with `-h` to see allowed flags. `ovnkube-observ` is installed on the ovnkube pods. For example:
 
-This feature requires OVS 3.4 and linux kernel 6.11.
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)